src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -128,16 +141,15 @@ class Autocomplete extends React.Component {
 
   render() {
     // eslint-disable-next-line no-unused-vars
-    const { id, className, value, onChange, errorText, clearable, creatable, ...rest } = this.props;
+    const { id, className, value, onChange, errorText, clearable, creatable, async, ...rest } = this.props;
 
     const classes = this._buildClass(className);
 
     const errorTextComponent = !_.isNil(errorText) ? (
       <div className="ui-128__autocomplete--error-text">{errorText}</div>
     ) : null;
 
-    const selectComponent = creatable ? TetheredCreatable : TetheredSelect;
-
+    const selectComponent = creatable ? (async ? TetheredAsyncCreatable : TetheredCreatable) : TetheredSelect;
```

Comment:
nit: Could use a function to encapsulate the logic `async ? TetheredAsyncCreatable : TetheredCreatable` to get rid of the nested conditional ternary.